docs/TuxReact.md

```diff
@@ -0,0 +1,449 @@
+# Tux/React
+
+## Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#React-Components-in-a-Nutshell">React Components in a Nushell</a>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Writing-React-and-tuxReact-in-JSX">Writing React and tuxReact in JSX</a></li>
+      <li><a href="#Rendering-in-React">Rendering in React</a></li>
+      <li><a href="#React-Components">React Components</a>
+        <ol style="list-style-type:upper-roman">
+          <li><a href="#React-Components-the-Render-Method">the Render Method</a></li>
+          <li><a href="#React-Components-Accessing-the-DOM-from-React">Accessing the DOM from React</a></li>
+          <li><a href="#React-Components-Props-and-State">Props and State</a>
+            <ol>
+              <li><a href="#React-Components-props-children">props.children</a></li>
+              <li><a href="#React-Components-PropTypes">PropTypes</a></li>
+            </ol>
+          </li>
+          <li><a href="#React-Components-Lifecycle-Methods">Lifecycle Methods</a></li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+  <li><a href="#TuxedoJS-and-React">TuxedoJS and React</a>
+    <ol>
+      <li><a href="#TuxedoJS-OwnerClass">TuxedoJS OwnerClass</a></li>
+      <li><a href="#TuxedoJS-OwneeClass">TuxedoJS OwneeClass</a></li>
+      <li><a href="#TuxedoJS-MutableClass">TuxedoJS MutableClass</a></li>
+    </ol>
```

Comment:
Do you want to have a link to the Conclusion? It is short, yet it might be good to have that link in the ToC.